### DISCUSSION TITLE
Exporting the simulation

### DISCUSSION THREAD

**Discussion by BoySku** [InteractiveComputerGraphics/SPlisHSPlasH#166] [Q&A] [answered]
Hello there ,
Firstly i would like to thank all the people working on this project, such an amazing tool.
my question is how can i export the simulation after finishing up the scene, and what formats are supported 
thanks

**Comment by janbender** [comment]
Currently we can export partio, VTK and OBJ files. However, it is quite simple to write own exporters.

**Reply by BoySku** [reply]
is there any tutorial on how to use the exported VTK or OBJ files with a rendering software (Blender)?

**Comment by janbender** [comment, marked as answer]
You can import the data in Paraview to visualize it. Moreover we are just implementing a new plugin for Blender to import the VTK and OBJ data there. For partio files we already published a Blender plugin.

**Reply by BoySku** [reply]
got it now thank you very much
